src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -329,7 +329,23 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		}, [this.listRenderer],
 			{
 				keyboardSupport: false,
-
+				accessibilityProvider: {
+					getAriaLabel: element => {
+						if (element.action instanceof CodeActionAction) {
+							let label = element.action.label;
+							if (!element.action.enabled) {
+								if (element.action instanceof CodeActionAction) {
+									label += `Disabled Reason: ${element.action.action.disabled}`;
```

Comment:
This needs to be localized. Also this likely needs a separator between the label and `Disabled Reason`